.github/workflows/main.yml

```diff
@@ -36,4 +36,4 @@ jobs:
         version: latest
         args: release --rm-dist
       env:
-        GITHUB_TOKEN: ${{ secrets.JASONBLANCHARD_TMP_REPO_RW }}
+        GITHUB_TOKEN: ${{ secrets.INSTRUCTURE_BRIDGE_GITHUB_BOT_REPO_RW }}
```

Comment:
I set this up as an org secret so any repo in the org should be able to access it